### DISCUSSION TITLE
Update API Gateway to more than 29 seconds

### DISCUSSION THREAD

**Discussion by luigi-castro** [serverless/serverless#12635] [Q&A] [answered]
Hi all!

Base on the latest documentation of AWS: https://aws.amazon.com/about-aws/whats-new/2024/06/amazon-api-gateway-integration-timeout-limit-29-seconds/

Gateway timeout can now be updated to more than 29 seconds.

But base on the serverless documentation for both API Gateway v1 and API Gateway v2 there is seems no key specific for the API Gateway timeout.

Is there are workaround for this other than editing the API Gateway properties on AWS, we currently don't have any permission to update API Gateway directly so that is not an option as of now.

Reference: 
https://www.serverless.com/framework/docs-providers-aws-events-apigateway
https://www.serverless.com/framework/docs-providers-aws-events-http-api

**Comment by czubocha** [comment, marked as answer]
The ability to set the AWS API Gateway integration timeout has been released in [v4.4.13](https://github.com/serverless/serverless/releases/tag/v4.4.13). You can find the docs [here](https://www.serverless.com/framework/docs/providers/aws/events/apigateway#timeout).
Feel free to share any further issues, feature requests, or feedback!
